docs/content/preview/architecture/docdb/persistence.md

```diff
@@ -50,8 +52,7 @@ DocumentKey1 = {
 }
 ```
 
-Keys stored in RocksDB consist of a number of components, where the first component is a document
-key, followed by several scalar components, and finally followed by a MVCC timestamp (sorted in reverse order). Each component in the DocumentKey, SubKey, and Value, are PrimitiveValues, which are type value pairs that can be encoded to and decoded from strings. When encoding primitive values in keys, a binary-comparable encoding is used for the value, so that sort order of the encoding is the same as the sort order of the value.
+Keys stored in RocksDB consist of a number of components, where the first component is a document key, followed by several scalar components, and finally followed by a MVCC timestamp (sorted in reverse order). Each component in the DocumentKey, SubKey, and Value, are PrimitiveValues, which are type value pairs that can be encoded to and decoded from strings. When encoding primitive values in keys, a binary-comparable encoding is used for the value, so that sort order of the encoding is the same as the sort order of the value.
```

Comment:
encoded to and decoded from strings -> encoded to and decoded from byte arrays

Reply:
:+1: